Kinvey.Shared/Auth/Credential.cs

```diff
@@ -81,39 +86,43 @@ public class Credential : IKinveyRequestInitializer
 		[DataMember]
 		private KinveyUserMetaData userKMD;
 
-		/// <summary>
-		/// The device ID associated with this user.
-		/// </summary>
-		[DataMember]
+        /// <summary>
+        /// The device ID associated with this user.
+        /// </summary>
+        /// <value>The DeviceID property gets/sets the value of the string field, _deviceID.</value>
+        [DataMember]
 		public string DeviceID { get; set; }
 
 		/// <summary>
 		/// Gets or sets the MIC Client identifier.
 		/// </summary>
-		/// <value>The MICC lient identifier.</value>
+		/// <value>The MIC client identifier.</value>
 		[DataMember]
 		public string MICClientID { get; set; }
 
 		/// <summary>
-		/// Initializes a new instance of the <see cref="KinveyXamarin.Credential"/> class.
+		/// Initializes a new instance of the <see cref="Credential"/> class.
 		/// </summary>
 		[Preserve]
 		public Credential()
 		{
 		}
 
-		/// <summary>
-		/// Initializes a new instance of the <see cref="KinveyXamarin.Credential"/> class.
-		/// </summary>
-		/// <param name="userId">User ID</param>
-		/// <param name="accessToken">Access token</param>
-		/// <param name="authToken">Auth token</param>
-		/// <param name="userName">User name</param>
-		/// <param name="attributes">User attributes</param>
-		/// <param name="kmd">Kinvey metadata</param>
-		/// <param name="refreshToken">Refresh token</param>
-		/// <param name="redirectURI">Redirect URI</param>
-		public Credential(string userId,
+        /// <summary>
+        /// Initializes a new instance of the <see cref="KinveyXamarin.Credential"/> class.
```

Comment:
as above, `KinveyXamarin.Credential` needs to be replaced by `Credential`

Reply:
@heyzooi  Thank you Victor for finding. I will fix it!